cms/migrations/0050_helper_text_change.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.18 on 2021-04-13 18:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0049_benefitspage_content_table_block'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='programpage',
+            name='program_subscribe_link',
+            field=models.URLField(blank=True, help_text='A url for Learn More button, if there is no course(s) available.', null=True),
```

Comment:
Why do we need a migration here?

If we must do a migration, we might as well change the help text, I think?